activedoptree.py

```diff
@@ -60,22 +100,22 @@ def __init__(self, ptree: ParentedTree, senttok: List[str], cgel_terminals = Non
 		ptree_terminals_with_labels = copy.deepcopy([subt for subt in self.ptree.subtrees(lambda t: t.height() == 2)])
 		# convert ptree to a CGELTree object
 		self.cgel_tree = self._ptree_to_cgel()
-		# update ptree by canonicalizing the position of punctuation terminals/preterminals
-		self.ptree = self._cgel_to_canonicalized_ptree()
+		# back-convert the CGELTree object to a ParentedTree object to canonicalize punctuation positions
+		self.ptree, self.senttok = cgel_to_ptree(self.cgel_tree)
 		# update canonicalized ptree with standardized labels
 		for i, subt in enumerate(self.ptree.subtrees(lambda t: t.height() == 2)):
 			subt.label = ptree_terminals_with_labels[i].label
 		# update cgel_tree with a set of terminals if provided
 		if cgel_terminals is not None:
 			self.cgel_tree.update_terminals(cgel_terminals, gaps=True, restore_old_cat=True, restore_old_func=True)
-	
+
 	def brackettreestr(self, pretty = False):
```

Comment:
Do we still need this?

Reply:
(I guess this question applies to all mentions of "bracket" in the code)

Reply:
We only need this for the legacy PTB edit mode, to write bracket tree strings to the text box. (We never use it to edit CGEL trees). I've been trying periodically to make sure that we still support editing in non-CGEL formats.